docs/modules/ROOT/partials/exoscale/environment-vars.adoc

```diff
@@ -1,9 +1,26 @@
 .Access to cloud API
 [source,bash]
 ----
-export EXOSCALE_ACCOUNT=<exoscale-account>
+
 export EXOSCALE_API_KEY=<exoscale-key>
 export EXOSCALE_API_SECRET=<exoscale-secret>
-export EXOSCALE_REGION=<exoscale-zone>
+export EXOSCALE_REGION=<exoscale-zone> <1>
+
+export EXOSCALE_CONFIG=`pwd`/exoscale.toml
 export EXOSCALE_S3_ENDPOINT="sos-${EXOSCALE_REGION}.exo.io"
+
+cat <<EOF > "$EXOSCALE_CONFIG"
+defaultaccount = "cluster-setup"
+
+[[accounts]]
+  name = "cluster-setup"
+
+  defaultZone = "${EXOSCALE_REGION}"
+  endpoint = "https://api.exoscale.ch/v1"
+  key = "${EXOSCALE_API_KEY}"
+  secret = "${EXOSCALE_API_SECRET}"
+EOF
+
+unset EXOSCALE_API_KEY EXOSCALE_API_SECRET
```

Comment:
Are you sure we do not need them when passing it to terraform?

Reply:
Looking at the installation guide, we need those environment variables for Terraform (e.g. step 1 in https://kb.vshn.ch/oc4/how-tos/exoscale/install.html#_provision_infrastructure).

Reply:
Crap 🙈 As soon as these are set the region is ignored 😅 I'll rename them...